go.mod

```diff
@@ -1,8 +1,8 @@
 module go.k6.io/k6
 
-go 1.21
+go 1.22
 
-toolchain go1.21.13
+toolchain go1.23.2
```

Comment:
Have we agreed on that change? Shouldn't be here, something like `toolchain go1.22` since we still use it CI?

Reply:
I've updated it to `go1.22.10` after our internal discussion.

[e3a7a1d](https://github.com/grafana/k6/pull/4177/commits/e3a7a1dfdebc44435220db84b7ec68bf8d4e35c0)